composer.json

```diff
@@ -6,28 +6,31 @@
     "require": {
         "php": "^7.1",
         "ext-soap": "*",
-        "php-http/client-common": "^1.6",
-        "php-http/discovery": "^1.3",
-        "php-http/guzzle6-adapter": "^1.0",
-        "php-http/httplug": "^1.0",
-        "php-http/message": "^1.6",
-        "php-http/message-factory": "^1.0",
+        "php-http/client-common": "^2",
```

Comment:
After discussing it with @voidtek we decided we can remove this.

Reply:
:+1: